appendix/basic-theory.tex

```diff
@@ -40,3 +40,47 @@ \subsection{Graph Convolutional Network}
 An issue with this propagation function could be that the value of each node now is a sum of each of its neighbors, and therefore loses its own value.
 This could be solved by replacing $A$ with $\hat{A} = A + I$ where $I$ is the identity matrix.
 Doing this the node considers itself a neighbor.
+
+\subsection{Performance metrics}
+Throughout this subsection we are describing NDCG, recall and precision.
+
+\subsubsection{NDCG}
+Normalized Discounted Cumulative Gain (NDCG) is a measure of ranking quality \cite{NDCG-evaluation}.
+To calculate the value of NDCG, it is necessary to have the Discounted Cumulative Gain (DCG).
+The equation for DCG is,
+\begin{equation}
+    DCG = \sum_{i=1}^{n} \frac{relevance_i}{log_2(i+1)},
+\end{equation}
+where $relevance_i$ is the actual recommendation value.
```

Comment:
hedder det en recommendation value? regner med det betyder sandsynligheden for at et item skal recommendes eller hvordan?